.circleci/config.yml

```diff
@@ -98,7 +98,7 @@ executors:
 
   node_js:
     docker:
-      - image: node:lts
+      - image: node:16
```

Comment:
node lts is v20 in the [docker registry](https://hub.docker.com/_/node). is this intentional?

Reply:
Ah I think I've been mis-reading the config. I'll remove these :)